docs/fidesops/docs/guides/saas_config.md

```diff
@@ -0,0 +1,256 @@
+## What is a SaaS configuration schema?
+
+A SaaS connector is defined in two parts, the [Dataset](dataset.md) and the SaaS config. The Dataset describes the data that is available from the connector and the SaaS config describes how to connect and retrieve/update the data in the connector. If you contrast this to a [database connector](database_connectors.md), the ways to retrieve/update data conform to a specification (such as SQL) and are consistent. When accessing data from APIs, each application or even different endpoints within the same application can follow different patterns. It was necessary to have a flexible configuration to be able to define the different access/update patterns.
+
+In short, you can think of the Dataset as the "what" (what data is available from this API) and the SaaS config as the "how" (how to access and update the data).
+
+#### An example SaaS config
+
+This config defines:
+
+- The domain and authentication requirements for an HTTP client to Mailchimp
+- A test request for verifying the connection was set up correctly
+- Endpoints to the following resources within the Mailchimp API:
+    - GET and UPDATE for the `members` resource
+    - GET for the `conversations` resource
+    - GET for the `messages` resource
```

Comment:
It might be useful to link to the corresponding Mailchimp API docs for each of these endpoints? it took me awhile to find them myself, and comparing their API docs with your dataset/saas config was helpful.

Reply:
Updated with links to the Mailchimp API